sherpa/utils/__init__.py

```diff
@@ -1022,36 +1029,60 @@ def filter_bins(mins, maxes, axislist):
     Examples
     --------
 
-    Calculate those points in xs which are in the range 1.5 <= x <= 2.3.
+    Calculate those points in xs which are in the range 1.5 <= x <= 4.
 
-    >>> mask = filter_bins([1.5], [2.3], [xs])
+    >>> xs = [1, 2, 3, 4, 5]
+    >>> filter_bins([1.5], [4], [xs])
+    array([False,  True,  True,  True, False])
 
     Repeat the above calculation by combining filters for x >= 1.5
-    and x <= 2.3.
+    and x <= 4 (note that the grid must be repeated for each
+    filter):
+
+    >>> filter_bins([1.5, None], [None, 4], [xs, xs])
+    array([False,  True,  True,  True, False])
+
+    For integrated data sets the lower and upper edges should be sent
+    separately with the max and min limits, alng with setting the
```

Comment:
```suggestion
    separately with the max and min limits, along with setting the
```

Reply:
Addressed in https://github.com/sherpa/sherpa/pull/1216/commits/73ca7cc38e045b2883205f5a11b497a90e8a49bf